Digizuite/Extensions/MetadataValueServiceExtensions.cs

```diff
@@ -73,19 +74,16 @@ public static class MetadataValueServiceExtensions
             field.FieldItemId = fieldResponse.ItemId;
             field.MetafieldId = fieldResponse.MetafieldId;
             field.LabelId = fieldResponse.LabelId;
-            field.Label = fieldResponse.Label;
-            field.LanguageId = fieldResponse.LanguageId;
```

Comment:
I think we need to keep both languageId and label. The response is used for the editor API call, and that need to know these things.